dask/array/tests/test_slicing.py

```diff
@@ -1,17 +1,19 @@
-import pytest
-pytest.importorskip('numpy')
-
 import itertools
 from operator import getitem
 
-from dask.compatibility import skip
-import dask.array as da
-from dask.array.slicing import (slice_array, _slice_1d, take, new_blockdim,
-                                sanitize_index)
-from dask.array.utils import assert_eq
 import numpy as np
+import pytest
 from toolz import merge
 
+import dask.array as da
+from dask.array.slicing import (_sanitize_index_element, _slice_1d,
+                                new_blockdim, sanitize_index, slice_array,
+                                take)
+from dask.array.utils import assert_eq
+from dask.compatibility import skip
+
+pytest.importorskip('numpy')
```

Comment:
This line should be left at the top of the file, or at least before the `import numpy as np` line. Or even better, merge the two lines as:
`np = pytest.importorskip('numpy')`